src/FileUploadControl.php

```diff
@@ -320,14 +323,16 @@ public function setUploadModel(string $uploadModel): self
 	/**
 	 * @internal
 	 */
-	public function getMaxFileSize(): int
+	public function getMaxFileSize(): ?int
 	{
 		return $this->maxFileSize;
 	}
 
-	public function setMaxFileSize(string $maxFileSize): self
+	public function setMaxFileSize(?string $maxFileSize): self
 	{
-		$this->maxFileSize = $this->parseIniSize($maxFileSize);
+		if (isset($maxFileSize)) {
```

Comment:
Proc muzu zadat maxFileSize = null?

Reply:
Vycházím z toho, že výchozí může být null (https://github.com/contributte/fileupload/blob/67b60d98ef096b2372415023c2bcda3d00c74a06/src/FileUploadExtension.php#L23) a zároveň parseIniSize (https://github.com/contributte/fileupload/blob/67b60d98ef096b2372415023c2bcda3d00c74a06/src/FileUploadControl.php#L568) má mít string.

Má tam být tedy spíše jen string nebo třeba ošetřit null jak v konstruktoru (https://github.com/contributte/fileupload/blob/67b60d98ef096b2372415023c2bcda3d00c74a06/src/FileUploadControl.php#L135)?